src/lib/list/list.scss

```diff
@@ -139,10 +147,20 @@ $mat-dense-list-icon-size: 20px;
       $mat-list-icon-size
     );
   }
+
+  .mat-list-option {
+    @include mat-line-base(
```

Comment:
Do you mean `mat-list-item-base` here? The `mat-line-base` mixin only takes one arg.